### ISSUE TITLE
Syntax validation for Java is not working 

### ISSUE THREAD

**Issue by yogitaAP** [angular-ui/ui-ace#162]
I am using ace editor in an angular application and I have initialized the editor in Java mode. The syntax highlighting for Java is not working. However, if I change the mode to javascript, it works fine. Why is that so?. Also, I set "$scope.editor.getSession().setUseWorker(true);" explicitly.

Attaching the screenshot for reference.
Java code

<img width="677" alt="screen shot 2016-12-05 at 12 08 46 pm" src="https://cloud.githubusercontent.com/assets/9781726/20876222/7091daf8-bae6-11e6-96ea-0f12dad62d76.png">

Javascript Code

<img width="678" alt="screen shot 2016-12-05 at 12 06 09 pm" src="https://cloud.githubusercontent.com/assets/9781726/20876228/75db980a-bae6-11e6-92dd-1270fb5d84aa.png">

**Comment by mmohdnowfal** [comment]
same issue in python mode too, is there any other settings to enable the validation

**Comment by bfaf** [comment]
For React only 10 languages have support for syntax validation. Java is not among them. I guess the same is true for Angular.

Pretty much the directories in [this repo](https://github.com/ajaxorg/ace/tree/master/lib/ace/mode) correspond to the languages that have syntax validation out of the box.

**Comment by sheshit** [comment]
@yogitaAP Have you tried integrating java syntax checking to ace editor in angular? Any inputs will be useful.

**Comment by yogitaAP** [comment]
Hey Sheshit,

Yes, I did integrate JAVA syntax checker. It worked.

**Comment by sheshit** [comment]
@yogitaAP How did you do it? I have tried adding my custom syntax checker for JAVA in the ace package. But couldn't find a linter file for JAVA. which linter have you used?

**Comment by yogitaAP** [comment]
Hey Sheshit,

Did it a long time ago? I think ace has built in methods to put syntax
checker on. Try searching it.

**Comment by sheshit** [comment]
No, ace doesn't provide support for JAVA. We need to write our own syntax checker as a web worker. Ok. Thanks Yogita.
